HISTORY.md

```diff
@@ -0,0 +1,655 @@
+# History
+
+History is important, but our current roadmap can be found [here](https://github.com/cookiecutter/cookiecutter/milestones?direction=desc&sort=due_date&state=open)
+
+## 1.7.0 (????-??-??) ???????
+
+Important Changes:
+
+* Drop support for EOL Python 3.4, thanks to [\@jamescurtin](https://github.com/jamescurtin) and [\@insspb](https://github.com/insspb) (\#1024)
```

Comment:
Do user references need to be escaped?  Looks like it works ok without this in AUTHORS.md?

Reply:
Ok, removed.